content/bn/virtualization.md

```diff
@@ -7,25 +7,18 @@ tags: ["fundamental", "infrastructure", "methodology"]
 
 ## এটা কি
 
-ভার্চুয়ালাইজেশন, ক্লাউড নেটিভ কম্পিউটিং প্রসঙ্গে,
-একটি শারীরিক কম্পিউটার নেওয়ার প্রক্রিয়াকে বোঝায়, কখনও কখনও একটি সার্ভার বলা হয়,
-এবং এটি একাধিক বিচ্ছিন্ন অপারেটিং সিস্টেম চালানোর অনুমতি দেয়।
-সেই বিচ্ছিন্ন অপারেটিং সিস্টেম এবং তাদের ডেডিকেটেড কম্পিউট রিসোর্স (সিপিইউ, মেমরি এবং নেটওয়ার্ক)
-ভার্চুয়াল মেশিন বা ভিএম হিসাবে উল্লেখ করা হয়।
-যখন আমরা একটি [ভার্চুয়াল মেশিন](/bn/virtual-machine/) সম্পর্কে কথা বলি, তখন আমরা একটি সফ্টওয়্যার-সংজ্ঞায়িত কম্পিউটারের কথা বলছি।
+ভার্চুয়ালাইজেশন, ক্লাউড নেটিভ কম্পিউটিং সম্পর্কে, <br>-কখনওকি কখনও একটি সার্ভারের সঙ্গে সম্প্রেষণ করে এবং এটি একাধিক বিভিন্ন অপারেটিং সিস্টেম চালাতে দেয়। <br>-এই বিভিন্ন অপারেটিং সিস্টেম এবং তাদের ডেডিকেটেড কম্পিউটিং সম্পদ (সিপিইউ, মেমরি, এবং নেটওয়ার্ক) <br>-ভার্চুয়াল মেশিন বা ভিএম হিসাবে পরিচিত হয়।
```

Comment:
Hello @PrashantDesale2004.

Kindly remove `<br>` tags. As you can see in the below picture that it is breaking the UI.
![image](https://github.com/cncf/glossary/assets/60180521/794bd662-fa24-4347-8c26-a6f2ac83a19f)
 
 Kindly make those sentence more easy for reader?